projects/languages/ros/webots_ros/src/complete_test.cpp

```diff
@@ -143,13 +143,16 @@ double GyroValues[3] = {0, 0, 0};
 double inertialUnitValues[4] = {0, 0, 0, 0};
 double touchSensorValues[3] = {0, 0, 0};
 
+bool callbackCalled = false;
```

Comment:
We should probably declare this variable as `static` and all the other ones above as well.

```suggestion
static bool callbackCalled = false;
```